docs/guides/10-add-legends.md

```diff
@@ -121,7 +121,7 @@ const viz = new carto.Viz(`
 Since the `ramp` expression is the root expression of a styling property (in this case, `color`) it can be accessed directly with `layer.viz.color.getLegendData()`.
 
 **Note:**
-If styling isn't tied directly to a styling property, you will need to use a variable. We will explore that method in detail in the [Add Widgets](/developers/carto-vl/guides/add-widgets/) guide.
+If styling isn't tied directly to a styling property, you will need to use a [variable](https://carto.com/help/glossary/#variable). We will explore that method in detail in the [Add Widgets](/developers/carto-vl/guides/add-widgets/) guide.
```

Comment:
I think we could use the more precise definition from CARTO VL and not the general glossary definition on this here. What do you think?

Reply:
ahhh ok, yes. missed that we had that in our glossary.